src/Tools/dotnet-user-jwts/src/Helpers/DevJwtCliHelpers.cs

```diff
@@ -0,0 +1,181 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Text.Json;
+using System.Xml.Linq;
+using System.Xml.XPath;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.UserSecrets;
+using Microsoft.Extensions.Tools.Internal;
+
+namespace Microsoft.AspNetCore.Authentication.JwtBearer.Tools;
+
+internal static class DevJwtCliHelpers
+{
+    public static string GetUserSecretsId(string projectFilePath)
+    {
+        var projectDocument = XDocument.Load(projectFilePath, LoadOptions.PreserveWhitespace);
+        var existingUserSecretsId = projectDocument.XPathSelectElements("//UserSecretsId").FirstOrDefault();
+
+        if (existingUserSecretsId == null)
+        {
+            return null;
```

Comment:
I've always disliked this about `user-secrets` -- I've told the tool my intent, if you can't find the secrets key, just create one for me, don't make me extra hop to 'init' it.  Let's do that here...if not found, create a new secrets id, add it to the project, output that created one as info ('No secrets file found, created with id {}')

Reply:
Rather than gate the check-in on these suggestions, I think we should log them in a separate issue that can be addressed in preview.6